railties/test/application/asset_debugging_test.rb

```diff
@@ -86,13 +86,17 @@ class ::PostsController < ActionController::Base ; end
       }
 
       cases.each do |(view_method, tag_match)|
-        app_file "app/views/posts/index.html.erb", "<%= #{view_method} '#{contents}', skip_pipeline: true %>"
+        app_file "app/views/posts/#{view_method}.html.erb", "<%= #{view_method} '#{contents}', skip_pipeline: true %>"
 
         app "development"
 
-        class ::PostsController < ActionController::Base ; end
+        class ::PostsController < ActionController::Base
+          def index
+            render params[:view_method]
+          end
+        end
 
-        get "/posts?debug_assets=true"
+        get "/posts?debug_assets=true&view_method=#{view_method}"
```

Comment:
If we do the above mentioned we can keep this the same as before.
```suggestion
        get "/posts?debug_assets=true"
```

Reply:
Thanks